scripts/docker/rhel.6/Dockerfile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM microsoft/dotnet-buildtools-prereqs:centos-6-783abde-20171304101322
```

Comment:
based on centos? Not rhel 6?

Reply:
Yes, CentOS 6 and RHEL 6 are equivalent w.r.t. to the versions of packages available for these.